pkg/stream/constants.go

```diff
@@ -8,6 +8,7 @@ const (
 const (
 	MessageTypeLogsConnect MessageType = iota
 	MessageTypeExecConnect
+	MessageTypeAttachConnect
```

Comment:
Here is a compatibility problem. We hope CloudCore can be compatible with EdgeCore of earlier versions generally.  If `MessageTypeAttachConnect` is inserted here, when CloudCore send `MessageTypeData`，EdgeCore will identify it as `MessageTypeRemoveConnect`. I suggest adding `MessageTypeAttachConnect` at the end, and the judgment at   line 195 of `edge/pkg/edgestream/tunnel` needs to be changed accordingly.

Reply:
fixed